# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gpu-descriptor/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gpu-descriptor/.cargo-checksum.json b/third_party/rust/gpu-descriptor/.cargo-checksum.json
--- a/third_party/rust/gpu-descriptor/.cargo-checksum.json
+++ b/third_party/rust/gpu-descriptor/.cargo-checksum.json
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"1dcee1269f28d755a95cd0349dccece43da0ba854ce7a3d199f67d256e7fb29a","src/allocator.rs":"6add0ad24e55c75861a2c2d8d7214cad3ba8aab0fd1750388554a941450f8cdd","src/lib.rs":"fb014819810bf90f8ef006f493c6f97f8fc43b01afa5e43faebee7b2b311c1b9"},"package":"e8a70f1e87a3840ed6a3e99e02c2b861e4dbdf26f0d07e38f42ea5aff46cfce2"}
\ No newline at end of file
+{"files":{"Cargo.toml":"8f600ea731067bfd91fd1aaadc2dad74fd681b10eb2955a19d2a308adc39873b","src/allocator.rs":"d46bee4586b88a3a5988e59b3112da379a49688d7f12bd2716cebb9fa6076a7a","src/lib.rs":"6fb74a08ad9975e561f4fca7bd391f0cbd96a7cab79b17df7d979021099b50f9"},"package":"d7a237f0419ab10d17006d55c62ac4f689a6bf52c75d3f38b8361d249e8d4b0b"}
\ No newline at end of file